android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TabSessionState.kt

```diff
@@ -36,6 +36,7 @@ data class TabSessionState(
     override val id: String = UUID.randomUUID().toString(),
     override val content: ContentState,
     override val trackingProtection: TrackingProtectionState = TrackingProtectionState(),
+    override val translationsStatus: TranslationsStatus = TranslationsStatus(),
```

Comment:
Please add @property for translationsStatus.

Reply:
Updated!